src/browser.js

```diff
@@ -204,20 +212,12 @@ function save(namespaces) {
  * @api private
  */
 function load() {
-	let r;
 	try {
-		r = exports.storage.getItem('debug');
+		return exports.storage.getItem('debug');
 	} catch (error) {
 		// Swallow
 		// XXX (@Qix-) should we be logging these?
 	}
-
-	// If debug isn't set in LS, and we're in Electron, try to load $DEBUG
-	if (!r && typeof process !== 'undefined' && 'env' in process) {
```

Comment:
This is the critical part that was exposing `process.env` in the browser.  It's been pulled out into the electron-specific flow.